api/config.py

```diff
@@ -232,7 +232,7 @@ class Configuration(CoreConfiguration):
             "label": _("A short description of this library"),
             "description": _("This will be shown to people who aren't sure they've chosen the right library."),
             "category": "Basic Information",
-            "level": CoreConfiguration.ALL_ACCESS
+            "level": CoreConfiguration.SYS_ADMIN_ONLY
```

Comment:
@adriana-alter Since you're now using the `CoreConfiguration` level constants, it's probably useful to get rid of the comments above (ll. 225-227), since the constants are pretty self-explanatory and the numbers there might get out of sync with the constants.

Reply:
Hi Tim!  The comments are still relevant (to me, at least!), because the front end uses the numbers.  (If the constants got out of sync with the numbers in the comments, that would probably mean they were also out of sync with the front end, i.e. that the functionality was broken from the user's perspective; I can't see it happening accidentally.)

Reply:
I see. Thanks for clarifying!